initialization/layout.go

```diff
@@ -1,17 +1,41 @@
 package initialization
 
-import "github.com/spacemeshos/post/config"
+import (
+	"fmt"
+
+	"github.com/spacemeshos/post/config"
+)
 
 type filesLayout struct {
+	From uint64
+	To   uint64
+
 	NumFiles          uint
 	FileNumLabels     uint64
 	LastFileNumLabels uint64
 }
 
-func deriveFilesLayout(cfg config.Config, opts config.InitOpts) filesLayout {
+func deriveFilesLayout(cfg config.Config, opts config.InitOpts) (*filesLayout, error) {
```

Comment:
filesLayout doesn't need to be a pointer, just return `filesLayout{}` in the error case.